RealEstate/apps/core/static/style.css

```diff
@@ -133,6 +133,36 @@ button.close{
 	padding-left:10%
 }
 
+.categoryBtn {
+    padding: 5px;
+    margin:10px 0px !important;
+    background-color: #fff;
+    border: 1px solid #ddd;
+    width: 100%;
+}
```

Comment:
Should also add an overflow:hidden, the text flows outside of the button when the window is small